diff --git a/node_modules/npm-solarsystem/index.js b/node_modules/npm-solarsystem/index.js
index cc4ef86..073707c 100644
--- a/node_modules/npm-solarsystem/index.js
+++ b/node_modules/npm-solarsystem/index.js
@@ -53,7 +53,7 @@ module.exports.getEarth = function Earth() {
 
 module.exports.getMars = function Mars() {
     const mars = {
-        image: 'https://www.nasa.gov/sites/default/files/thumbnails/image/edu_what_is_mars.jpg',
+        image: 'https://science.nasa.gov/wp-content/uploads/2024/03/pia04304-mars.jpg',
         distanceFromSun: '145.925 million miles',
         yearLength: '687 Earth Days',
         description: 'The fourth planet from the Sun, Mars is a dusty, cold, desert world with a very thin atmosphere. This dynamic planet has seasons, polar ice caps and weather and canyons and extinct volacanoes, evidence of an even more active past.',
@@ -68,7 +68,7 @@ module.exports.getMars = function Mars() {
 
 module.exports.getJupiter = function Jupiter() {
     const jupiter = {
-        image:'https://www.nasa.gov/sites/default/files/thumbnails/image/hs-2016-24-a-print-new.jpg',
+        image:'https://science.nasa.gov/wp-content/uploads/2024/03/jupiter-marble-pia22946.jpg',
         distanceFromSun: '485.316 million miles',
         yearLength: '4333 Earth Days',
         description: 'Jupiter is the fifth planet from our Sun and is, by far, the largest planet in the solar system – more than twice as massive as all the other planets combined. Jupiters stripes and swirls are actually cold, windy clouds of ammonia and water, floating in an atmosphere of hydrogen and helium. Jupiter’s iconic Great Red Spot is a giant storm bigger than Earth that has raged for hundreds of years. Jupiter is surrounded by dozens of moons. Jupiter also has several rings, but unlike the famous rings of Saturn, Jupiter’s rings are very faint and made of dust, not ice.',
@@ -98,7 +98,7 @@ module.exports.getSaturn = function Saturn() {
 
 module.exports.getUranus = function Uranus() {
     const uranus = {
-        image: 'https://images.ctfassets.net/cnu0m8re1exe/7hQBr1Got3iOOxCn4yJqSy/8ac59ceb4d08d0474a2688ac976b1693/neptune1-1024x727.jpg?w=650&h=433&fit=fill',
+        image: 'https://upload.wikimedia.org/wikipedia/commons/6/69/Uranus_Voyager2_color_calibrated.png',
         distanceFromSun: '1.833 billion miles',
         yearLength: '30687 Earth Days',
         description: 'The seventh planet from the Sun with the third largest diameter in our solar system, Uranus is very cold and windy. The ice giant is surrounded by 13 faint rings and 27 small moons as it rotates at a nearly 90-degree angle from the plane of its orbit. This unique tilt makes Uranus appear to spin on its side, orbiting the Sun like a rolling ball. ',
